esnext-browsers.js

```diff
@@ -284,6 +284,18 @@ module.exports = {
       short: 'CH 52',
       note_id: 'experimental-flag',
   },
+  chrome53: {
```

Comment:
@ljharb, I have to disagree.  There's no data for Chrome 52 or Chrome 53.  You do not create granularity by creating empty columns of data.